### ISSUE TITLE
It doesn't work on my computer,

### ISSUE THREAD

**Issue by dongyoxuan1514** [IMXNOOBX/cs2-external-esp#8]
It doesn't work after compiling and running. I still ran it as an administrator. The game is also in full-screen window. Windows version is home version 22h2，Does it have anything to do with it?

**Comment by IMXNOOBX** [comment]
The Game has been updated, you have to update the offsets

**Comment by xXxNozz4rrxXx** [comment]
> It doesn't work after compiling and running. I still ran it as an administrator. The game is also in full-screen window. Windows version is home version 22h2，Does it have anything to do with it?



{
    "dwEntityList": 24692904,
    "dwLocalPlayer": 25664536,
    "dwPawnHealth": 2056,
    "dwPlayerPawn": 2044,
    "dwSanitizedName": 1824,
    "dwViewMatrix": 25667328,
    "m_iTeamNum": 959,
    "m_vecOrigin": 4628
    }

do not put fullscreen , put window fullscreen and it will work

**Comment by VilotStar** [comment]
> do not put fullscreen , put window fullscreen and it will work

Not quite sure what you mean lol, could you rephrase?

I haven't seen any issues with fullscreen or not fullscreen
